clang/test/Sema/warn-compare-enum-types-mismatch.c

```diff
@@ -6,7 +6,9 @@ typedef enum EnumA {
 } EnumA;
 
 enum EnumB {
-  B
+  B,
```

Comment:
It might be good to capture some of what I wrote in the review summary as a comment here so it's clear why this is correct in both C and C++.

Reply:
done

Reply:
I think what I was asking, was do we have an equivalent C++ test that verifies in a `.cpp` file that we also do not obtain a diagnostic for this.